voxblox/src/core/esdf_map.cc

```diff
@@ -4,7 +4,7 @@ namespace voxblox {
 
 bool EsdfMap::getDistanceAtPosition(const Eigen::Vector3d& position,
                                     double* distance) const {
-  constexpr bool interpolate = false;
+  constexpr bool interpolate = true;
```

Comment:
should this be exposed or do we always want to interpolate?

Reply:
I'll expose it, especially because of the performance difference you mentioned...